### DISCUSSION TITLE
Live/Uptodate List of all available actions and probes?

### DISCUSSION THREAD

**Discussion by xpdable** [chaostoolkit/chaostoolkit#202] [Q&A]
guys, while I am summarizing our chaos experiments. One question bothered me that is there any place that maintaining a list/file that showing all available actions and probes among all chaostoolkit and chaostoolkit extensions.

What I tried now is manually grabbed my used actions and probes into json. But it is not enough.. 
It is not up-to-date..
It is not for all extentsions..

Any idea?

**Comment by Lawouach** [comment]
Hello :+1: 
It's a useful feature indeed. Right now the docs listmost of them but it's a manual update and not available to the CLI.

**Reply by xpdable** [reply]
would it be possible to require all extension repos to keep an up-to-date file in an agreed configuration file json/md/yaml etc. Then chaostoolkit docs somehow fetch and aggregate all possible actions/probes from those files.

**Comment by Lawouach** [comment]
I just also realised this has nothing to do with the original issue question. Please open a different issue, or better come to our slack to discuss it. I'll remove the last messages from this thread to keep it clean.

**Reply by anu-sikki** [reply]
can you please help me with your slack link ?

**Reply by Lawouach** [reply]
https://join.chaostoolkit.org/ :)

**Reply by xpdable** [reply]
Hey Sylvain. I am doing other things busy this year. Hope I will manage some time and effort back on chaos 2022. Catch you @ Slack :)
